tests/postgresql_config/2018-04-04-softlayer-snapshot/inventory.ini

```diff
@@ -1,5 +1,5 @@
-[db0]
-10.162.36.205
+[db1]
+10.162.36.238
```

Comment:
I don't think you need to change this, since this is just a test based on a snapshot of how things were at the time it was created.